File: app/src/main/java/com/example/samson/immvrse/LogInActivity.java
package com.example.samson.immvrse;

import android.app.Activity;
import android.content.Intent;
import android.databinding.DataBindingUtil;
import android.graphics.Typeface;
import android.os.Bundle;
import android.text.method.PasswordTransformationMethod;
import android.view.View;

import com.example.samson.immvrse.databinding.ActivityLogInBinding;

public class LogInActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_log_in);
        ActivityLogInBinding binding = DataBindingUtil.setContentView(this, R.layout.activity_log_in);

        binding.signUp.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(LogInActivity.this, SignUpActivity.class);
                startActivity(intent);
            }
        });

        binding.signInButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(LogInActivity.this, OnBoardingActivity.class);
                startActivity(intent);
            }
        });

        binding.password.setTypeface(Typeface.DEFAULT);
        binding.password.setTransformationMethod(new PasswordTransformationMethod());
    }
}

File: app/src/main/java/com/example/samson/immvrse/LogInActivity.kt
package com.example.samson.immvrse

import android.content.Intent
import android.graphics.Typeface
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.text.method.PasswordTransformationMethod
import kotlinx.android.synthetic.main.activity_log_in.*

class LogInActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_log_in)

        sign_up.setOnClickListener {
            startActivity(Intent(this, SignUpActivity::class.java))
        }

        sign_in_button.setOnClickListener {
            startActivity(Intent(this, OnBoardingActivity::class.java))
        }

        password.typeface = Typeface.DEFAULT
        password.transformationMethod = PasswordTransformationMethod()
    }
}